### DISCUSSION TITLE
Landing page section in sidebar

### DISCUSSION THREAD

**Discussion by kai-tub** [pradyunsg/furo#567] [Questions] [answered]
First, thanks again for providing such an awesome Sphinx theme. :heart: 
This theme is literally the only reason why I _use_ Sphinx (with MyST).
Currently, one can only access the landing page (https://pradyunsg.me/furo/) by clicking on the logo/title.
I personally (and a few colleagues) find this unintuitive and can never _find_ the landing page again after looking at different sub-pages, as one has to _learn_ how to access the landing page by clicking on the logo.
What I would like is to have an extra link inside of the sidebar that redirects to the landing page.
I don't mind that one can access the landing page via clicking on the logo/title but I really would like to have a separate entry in the sidebar to have an accessible link to the landing page.

This behavior is similar to the one used in https://myst-nb.readthedocs.io/en/latest/# for example

Feel free to just point to a couple of links if I should've been able to figure that one out on my own :D 

Thanks!

**Comment by pradyunsg** [comment, marked as answer]
You can do this by adding `self` to the toctree in the index.md file.

For example, in Furo's own source code, the following change:

`````diff
diff --git a/docs/index.md b/docs/index.md
index 14c17bce..45d2c6db 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -14,6 +14,7 @@ # Furo
 ```{toctree}
 :hidden:
 
+Overview <self>
 quickstart
 customisation/index
 reference/index
`````

Results in:

<img width="1038" alt="Screenshot 2022-10-31 at 13 00 25" src="https://user-images.githubusercontent.com/3275593/199077270-fa3694fb-4282-4cd7-8021-fddc369322cf.png">

**Reply by kai-tub** [reply]
Thanks!
For future reference, this special key-word is defined by Sphinx: 
- https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-toctree

At the time of writing, it doesn't seem possible to do this with the [sphinx-external-toc](https://sphinx-external-toc.readthedocs.io/en/latest/intro.html) extension, see [issue 86](https://github.com/executablebooks/sphinx-external-toc/issues/86)
